torch/_meta_registrations.py

```diff
@@ -2716,4 +2716,9 @@ def activate_meta():
                 _meta_lib_dont_use_me_use_register_meta.impl(op_overload, fn)
 
 
+@register_meta(aten.all_reduce)
```

Comment:
what is this needed for again?

Reply:
AFAICT, this is needed to enable symbolic tracing by dynamo. @wconstab knows it better.

Reply:
yea. if we don't have this, then during tracing the 'real' kernel will get executed w/ fake inputs.  it won't work.